it/src/main/java/org/corfudb/universe/scenario/action/AddNodeAction.java

```diff
@@ -0,0 +1,29 @@
+package org.corfudb.universe.scenario.action;
+
+import lombok.Data;
+import org.corfudb.runtime.view.Layout;
+import org.corfudb.universe.node.CorfuServer;
+import org.corfudb.universe.scenario.action.Action.AbstractAction;
+import org.corfudb.universe.service.Service;
+
+/**
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Remove this unused import 'java.util.Optional'. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AUselessImportCheck)